docs/userGuide/syntax/links.mbdf

```diff
@@ -83,12 +85,14 @@ Within `index.md`, we can also display the image using
 <img src="textbook/image.png" />
 ```
 or by including `subsite.md`:
-```
+```html
 <include src="textbook/subsite.md" />
 ```
 
 <box type="important">
-To ensure that links in the _markbind/ folder work correctly across the entire site, they should be written as absolute paths, prepended with <span>{</span>{ baseUrl }}. 
+    Relative links to resources (e.g. images, hrefs) should be specified relative to the original, included file. 
+    <br>
+    In the example above, <code>image.png</code> is in the same directory as <code>subsite.md</code>. When using relative references, the correct path is <code>image.png</code> and not <code>textbook/image.png</code>.
```

Comment:
Should we be more explicit here? 

E.G: When using relative references in `subsite.md` after being included in `index.md`, ...

Reply:
Hmm, I intended to say something similar to
"When using relative references in subsite.md, links should be valid from the perspective of subsite.md, regardless of whichever file (such as `index.md`) that eventually includes it."